core/src/test/scala/integration/kafka/api/AuthorizerIntegrationTest.scala

```diff
@@ -143,6 +143,7 @@ class AuthorizerIntegrationTest extends BaseRequestTest {
   val adminClients = Buffer[Admin]()
 
   producerConfig.setProperty(ProducerConfig.ACKS_CONFIG, "1")
+  producerConfig.setProperty(ProducerConfig.ENABLE_IDEMPOTENCE_CONFIG, "false")
```

Comment:
Could we instead drop acks=1? I don't think it is important in this test.

Reply:
Did you miss this comment?

Reply:
Sorry, I forgot to reply this comment. No we should not remove the `acks=1` config to enable the idempotence producer. In this `AuthorizerIntegrationTest` test suite, we tested normal producer (created here), and IdempotentProducer (created in `buildIdempotentProducer` method), and transactional producer (created in `buildTransactionalProducer` method), so if we remove `acks=1` here, we won't be able to test normal producer cases.